libs/zk/include/nil/crypto3/zk/commitments/polynomial/lpc.hpp

```diff
@@ -421,6 +523,60 @@ namespace nil {
                         eval_storage_type z;
                         typename basic_fri::proof_type fri_proof;
                     };
+
+                    // Represents an initial proof, which must be created for each of the N provers.
+                    struct initial_proof_type {
```

Comment:
Maybe `lpc_proof_type`? Name looks like FRI initial proof.

We run lpc once for each prover and this proof will contain all information that is recieve from one ordinary prover.
* z
* initial_fri_proof(s)